source/nodejs/adaptivecards/src/controls/popup-menu.ts

```diff
@@ -21,6 +21,11 @@ export class PopupMenu extends PopupControl {
 
             element.appendChild(renderedItem);
 
+            if (i == 0)
+            {
+                renderedItem.setAttribute("aria-expanded", "true");
+            }
+
             if (i === this.selectedIndex) {
```

Comment:
nit: I'm wondering if we should set "aria-expanded" on the focused element?

Also FWIW, we might want to revisit this again later, because expanded is announced each time this menu item receives focus. I'm not sure it should be announced while navigating the list, only when the menu is first expanded.

Reply:
@anna-dingler ,

navigating using keys up and down will remove the aria-expanded. when the menu item is rendered, we set the first item to be in focus.

Reply:
Interesting, only commented on it because I saw the behavior on the test site. Not sure why, but it works correctly in regular mode, and doesn't work in scan mode.

Not blocking or anything though :)

Reply:
@anna-dingler, thanks, i will sync up with you offline.